README.md

```diff
@@ -37,6 +37,15 @@ environment variables, or with a link to a configuration guide.
 | AWS_TLS_INSECURE_SKIP_VERIFY | false                      | This should never be set to true, as it disables SSL certificate verification. Used only for development           
 | BIND_ADDR                    | :23900                     | The host and port to bind to                                                                                       
 | ELASTIC_SEARCH_URL           | "<http://localhost:11200>" | Http url of the ElasticSearch server                                                                               
+| -------------------          | -------------------       | -----------------------------------
+| NLP_BERLIN_API_ENDPOINT      | "/v1/berlin/search"       | Http url of the Berlin server
+| NLP_BERLIN_API_URL	         | "http://localhost:28900"  | Http url of the Berlin server
```

Comment:
whitespace issues

Reply:
There are whitespace issues because the line itself is too long and the editors usually put it on the next line so that you can see it, but it's still on one line. If you open it in viewer mode or check it in the repo it's rendered correctly.
This is sort of out of my control 
![Screenshot](https://github.com/ONSdigital/dp-search-api/assets/97289589/2a479825-333d-442e-9f41-b7fddb24e8b0)